backend/src/app.rs

```diff
@@ -1082,6 +1083,34 @@ mod test {
         assert!(res.is_err());
     }
 
+    #[tokio::test]
+    #[tracing_test::traced_test]
+    async fn test_event_create_email_fail_validation() {
+        let app = App::new(
+            Arc::new(InMemoryEventsDB::default()),
+            Arc::new(PubSubInMemory::default()),
+            Arc::new(MockViewers::new()),
+            Arc::new(Payment::default()),
+            Tracking::default(),
+            String::new(),
+        );
+
+        let res = app
+            .create_event(AddEvent {
+                data: EventData {
+                    name: String::from("123456789"),
+                    description: String::from("123456789 123456789 123456789 !"),
+                    short_url: String::new(),
+                    long_url: None,
+                },
+                moderator_email: "a@a",
```

Comment:
lets also have one test that proofs a valid-non-empty email works

Reply:
Will add now